tests/manage/storageclass/test_create_multiple_sc_with_same_pool_name.py

```diff
@@ -0,0 +1,129 @@
+import pytest
+import logging
+from tests import helpers
+from ocs_ci.ocs import constants
+from ocs_ci.ocs import defaults
+from ocs_ci.framework.testlib import ManageTest, tier1
+from tests.fixtures import (
+    create_ceph_block_pool,
+    create_rbd_secret, create_cephfs_secret,
+    create_project
+)
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@pytest.mark.usefixtures(
+    create_project.__name__,
+    create_rbd_secret.__name__,
+    create_cephfs_secret.__name__,
+    create_ceph_block_pool.__name__,
+)
+class TestCreateMultipleScWithSamePoolName(ManageTest):
+    """
+    Create Multiple Storage Class with same pool name
+    """
+    @pytest.mark.parametrize(
+        argnames="interface",
+        argvalues=[
+            pytest.param(
+                *["RBD"], marks=pytest.mark.polarion_id("OCS-622")
+            ),
+            pytest.param(
+                *["CEPHFS"], marks=pytest.mark.polarion_id("OCS-623")
+            )
+        ]
+    )
+    def test_create_multiple_sc_with_same_pool_name(self, interface):
+        """
+        This test function does below,
+        *. Creates multiple Storage Classes with same pool name
+        *. Creates PVCs using each Storage Class
+        *. Mount each PVC to an app pod
+        *. Run IO on each app pod
+        """
+        # Create 5 Storage Classes with same pool name
+        sc_obj_list = []
+        sc_list = []
+        for i in range(1, 3):
```

Comment:
Number of storage is given as 5 in previous comment and creating 2. Change accordingly

Reply:
Done

Reply:
Done